app/src/main/java/com/example/moodtrackerproject/data/NoteBody.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.moodtrackerproject.data
+
+import com.squareup.moshi.JsonClass
+import java.io.Serializable
+// @serializable
+@JsonClass(generateAdapter = true)
+data class NoteBody(
+    var date: String = "",
+    val title: String = "",
+    val text: String = " "
+) : Serializable
```

Comment:
Why Serializable?

Reply:
because Click fun will pass note in bundle to NoteDetails (??) as serialized **bundle.putSerializable("note", note)**